src/core/public/legacy_platform/legacy_platform_service.ts

```diff
@@ -54,6 +58,23 @@ export class LegacyPlatformService {
     bootstrapModule.bootstrap(this.params.rootDomElement);
   }
 
+  public stop() {
+    const angularRoot = angular.element(this.params.rootDomElement);
+    const injector$ = angularRoot.injector();
+
+    // if we haven't gotten to the point of bootstraping
+    // angular, injector$ won't be defined
+    if (!injector$) {
+      return;
+    }
+
+    // destroy the root angular scope
+    injector$.get('$rootScope').$destroy();
+
+    // clear the inner html of the root angular element
+    angularRoot.html('');
```

Comment:
__optional nit:__ `.text('')` maybe? I remember the days when vulnerability scanners were complaining about every `html()` occurrence :)